packages/headless/src/database/mongodb/tags/index.ts

```diff
@@ -17,4 +21,13 @@ export class TagsDatabase implements ITagsDatabase {
     const tag = await TagsModel.create(data);
     return tag?.toObject();
   }
+
+  async createHomePageProductsTags(data: HomePageProductsTagsRequest): Promise<Tag | null> {
+    const tag = await TagsModel.create(data);
```

Comment:
as the function name refers that you will create tags for home page only so add that flag here (hard-coded) so that no one can mistakenly use this function to create non-homepage tags.
actually the purpose of a function should be the same as the name refers.
you are creating home page tags by a generic function. because it takes generic data and just inserts.
the function will work if i pass isHomePageProductsTags: true. but that shouldn't be, as the function name refers.